### ISSUE TITLE
A reboot should not clear faillock by defalt

### ISSUE THREAD

**Issue by CoderThomasB** [linux-pam/linux-pam#454]
I noticed that if you are lockout of your account by faillock. Simply switching to the display manager (Ctrl + Alt + F1) or pressing the switch user button on the Gnome screen saver allows the user to reboot the computer and clear the faillock. If the reboot is fast enough it makes it easier to force the password if you have a small list of posable passwords. if the power button is configured to shutdown the computer and pressing it again turns it back on then there is another way of clearing faillock.

**Comment by t8m** [comment]
If you have a physical access to a device there are most probably plenty of other ways how to hack-into the device (unless you use disk encryption). I do not think this is a major concern.

You can also configure faillock to use a directory that is not cleared up on reboot.

**Comment by CoderThomasB** [comment]
> If you have a physical access to a device there are most probably plenty of other ways how to hack-into the device

Yes, if you have physical access an advanced attacker could gain access anyway. But I'm talking about someone that is not tech-savvy and is just typing the 20 most popular passwords by hand.
